src/software/proto/primitive/primitive_msg_factory.cpp

```diff
@@ -2,145 +2,152 @@
 
 #include "software/proto/message_translation/tbots_protobuf.h"
 
-std::unique_ptr<PrimitiveNewMsg> createChipPrimitiveMsg(const Point &chip_origin,
-                                                        const Angle &chip_direction,
-                                                        float chip_distance_meters)
+std::unique_ptr<TbotsProto::PrimitiveNew> createChipPrimitive(const Point &chip_origin,
+                                                              const Angle &chip_direction,
+                                                              double chip_distance_meters)
```

Comment:
just wondering, why did we change these from `float` to `double`

Reply:
All callers of these functions will be passing in `double`s (~99% of src/software uses `double`s instead of `float`s), so this saves the callers a `static_cast<float>`